libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -333,8 +350,20 @@ bool AP_Arming::battery_checks(bool report)
             }
             return false;
         }
-    }
 
+        if ((_min_voltage[0] > 0.0f) && (_battery.voltage(0) < _min_voltage[0])) {
+            if (report) {
+                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "PreArm: Battery 0 voltage below minimum");
```

Comment:
Agreed, it's just that incrementing the constant won't create the param
that's needed. But that's a concern either way. (And probably could be
solved with some macro work)
On Mar 2, 2016 12:33 PM, "Tom Pittenger" <notifications@github.com> wrote:

> In libraries/AP_Arming/AP_Arming.cpp
> <https://github.com/diydrones/ardupilot/pull/3604#discussion_r54777411>:
>
> >
> > +        if ((_min_voltage[0] > 0.0f) && (_battery.voltage(0) < _min_voltage[0])) {
> > +            if (report) {
> > +                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "PreArm: Battery 0 voltage below minimum");
>
> Which is why it should be using AP_BATT_MONITOR_MAX_INSTANCES. hard coded
> indexes are just as bad
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot/pull/3604/files#r54777411>.
>